src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -71,37 +73,26 @@ export const addAggregatorAndDestinationToMemo = ({
     'expected finalAssetLimitWithManualSlippage to be a positive amount',
   )
 
-  const aggregatorLastTwoChars = aggregator.slice(aggregator.length - 2, aggregator.length)
-  const finalAssetAddressLastTwoChars = finalAssetAddress.slice(
-    finalAssetAddress.length - 2,
-    finalAssetAddress.length,
-  )
-  const shortenedNativeAssetName =
-    shortenedNativeAssetNameByNativeAssetName[
-      nativeAssetName as keyof typeof shortenedNativeAssetNameByNativeAssetName
```

Comment:
@NeOMakinG we can keep the aggregator and L1 shorthands, this is absolutely safe and saves some memo space - just revert the last two chars bits for final assets

Reply:
I did revert it then 😆